doc/sphinx/language/gallina-extensions.rst

```diff
@@ -2005,10 +1996,14 @@ in :ref:`canonicalstructures`; here only a simple example is given.
   :undocumented:
 
 .. cmd:: Canonical {? Structure } @by_notation
-   :name: Canonical Structure X
+   :name: _
 
-   This command declares :token:`qualid` as a canonical instance of a
-   structure (a record).
+   The first form of this command declares an existing :n:`@qualid` as a
+   canonical instance of a structure (a record).
```

Comment:
This description makes no sense (in the current state) because you applied it without applying the changes to the command syntax that I suggested.

Reply:
Already fixed.

Also note the comments in the Conversation section are time-ordered.  I addressed this point a couple hours ago in two earlier comments.  Doesn't make it any easier.  And so many details.